### ISSUE TITLE
Admin-Interface has issues with non _system apps

### ISSUE THREAD

**Issue by wirtsi** [arangodb/arangodb#1255]
Hello

I'm following the guesser tutorial but I want to save the collections in my own database.

So instead of creating a symlink from
databases/_system/guesser -> ../../ 
I created
databases/cms/guesser -> ../../

When selecting the db in the web interface and then going to Apps, I can see that arango has picked up the app (dev mode), but it fails to properly load it there
```
Sorry the code is not documented properly
"Unable to read api 'cms' from path http://10.0.3.95:8529/_db/_system/_admin/aardvark/docu/dev%3Aguesser%3Acms/dev/cms?api_key=false (server returned Internal Error)"
```
Accessing the app through
http://10.0.3.95:8529/_db/cms/dev/cms/hello works fine so just seems to be a web interface issue.

When I add the symlink to _system/guesser the error goes away

Bests

wirtsi

**Comment by mchacki** [comment]
Hi Wirtsi,

thanks a lot for reporting this.
Which version of ArangoDB are you using?

**Comment by wirtsi** [comment]
Hi .. i'm using 2.4.3

**Comment by mchacki** [comment]
I will have a look into it.

**Comment by mchacki** [comment]
I can confirm it is broken in 2.4.3, it is already fixed in our  2.4 branch and will be included in 2.4.4.
Thanks a lot for reporting.
